src/test/java/org/sagebionetworks/bridge/exporter/handler/SynapseExportHandlerTest.java

```diff
@@ -583,6 +586,13 @@ public static void validateTsvRow(String line, String... extraValueVarargs) {
         dataGroupsDefinition.setTransferMethod(TransferMethod.STRINGSET);
         columnDefinitionsBuilder.add(dataGroupsDefinition);
 
+        ColumnDefinition substudyMembershipsDefinition = new ColumnDefinition();
+        substudyMembershipsDefinition.setName("substudyMemberships");
+        substudyMembershipsDefinition.setDdbName("userSubstudyMemberships");
+        substudyMembershipsDefinition.setMaximumSize(250L);
+        substudyMembershipsDefinition.setTransferMethod(TransferMethod.STRING);
+        columnDefinitionsBuilder.add(substudyMembershipsDefinition);
```

Comment:
Update this to match ColumnDefinition.json once that's been updated.

Reply:
Updated tests.